src/runtime/entity.ts

```diff
@@ -369,7 +368,7 @@ export abstract class Entity implements Storable {
   }
 
   static identify(entity: Entity, identifier: string, storageKey: string, creationTimestamp?: string) {
-    getInternals(entity).identify(identifier, storageKey, creationTimestamp);
+    getInternals(entity).identify(identifier, storageKey, new Date(Number(creationTimestamp)));
```

Comment:
will this work is creationTimestamp is null? or maybe we want to make the parameter non nullable?

Reply:
added a null check